chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/configuration/ChaosMonkeyRequestScopeSettingsTest.java

```diff
@@ -15,98 +15,62 @@
  */
 package de.codecentric.spring.boot.chaos.monkey.configuration;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.Test;
 
 /** @author Benjamin Wilms */
 class ChaosMonkeyRequestScopeSettingsTest {
 
-    private ChaosMonkeySettings settings;
-
     @Test
     void noArgsTest() {
-        settings = new ChaosMonkeySettings();
         ChaosMonkeyProperties chaosMonkeyProperties = getChaosMonkeyProperties();
-        settings.setChaosMonkeyProperties(chaosMonkeyProperties);
         AssaultProperties assaultProperties = getAssaultProperties();
-        settings.setAssaultProperties(assaultProperties);
         WatcherProperties watcherProperties = getWatcherProperties();
+        ChaosMonkeySettings settings = new ChaosMonkeySettings();
+        settings.setChaosMonkeyProperties(chaosMonkeyProperties);
+        settings.setAssaultProperties(assaultProperties);
         settings.setWatcherProperties(watcherProperties);
 
-        validate(chaosMonkeyProperties, assaultProperties, watcherProperties);
+        validate(settings, chaosMonkeyProperties, assaultProperties, watcherProperties);
     }
 
     @Test
     void allArgsTest() {
         ChaosMonkeyProperties chaosMonkeyProperties = getChaosMonkeyProperties();
         AssaultProperties assaultProperties = getAssaultProperties();
         WatcherProperties watcherProperties = getWatcherProperties();
-        settings = new ChaosMonkeySettings(chaosMonkeyProperties, assaultProperties, watcherProperties);
-
-        validate(chaosMonkeyProperties, assaultProperties, watcherProperties);
-    }
-
-    @Test
-    void lombokDataTest() {
-        ChaosMonkeyProperties chaosMonkeyProperties = getChaosMonkeyProperties();
-        AssaultProperties assaultProperties = getAssaultProperties();
-        WatcherProperties watcherProperties = getWatcherProperties();
-        settings = new ChaosMonkeySettings(chaosMonkeyProperties, assaultProperties, watcherProperties);
+        ChaosMonkeySettings settings = new ChaosMonkeySettings(chaosMonkeyProperties, assaultProperties, watcherProperties);
 
-        assertThat(settings.getChaosMonkeyProperties()).isNotNull();
```

Comment:
Not really sure, why this test exists, however dont we need these notnull checks still in a test (allArgsTest?)?